### PR TITLE
FE: updated routes and modal for create alert css

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Before: 

Screenshot 2024-02-27 at 2 36 49 AM

After: 

Dark mode: 

Screenshot 2024-02-27 at 2 43 36 AM

Light mode: 
Screenshot 2024-02-27 at 2 42 01 AM

## Summary by CodeRabbit

- **New Features**
    - Updated application routing to reflect a more organized structure for logs and traces views.
    - Introduced a new styling for the create alert modal, including support for a light mode.

### CODE REVIEW

**Review by Vikrant2520 — APPROVED** [review]

File: frontend/src/constants/routes.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ const ROUTES = {
 	TRACE_EXPLORER: '/trace-explorer',
 	BILLING: '/billing',
 	SUPPORT: '/support',
-	LOGS_SAVE_VIEWS: '/logs-save-views',
-	TRACES_SAVE_VIEWS: '/traces-save-views',
+	LOGS_SAVE_VIEWS: '/logs/saved-views',
+	TRACES_SAVE_VIEWS: '/traces/saved-views',
```
Comment (YounixM) [inline frontend/src/constants/routes.ts:46]: Please add old - new route mapping to handle auto redirects for saved / old routes

File: frontend/src/constants/routes.ts [hunk 2]
```diff
@@ -42,8 +42,8 @@ const ROUTES = {
 	TRACE_EXPLORER: '/trace-explorer',
 	BILLING: '/billing',
 	SUPPORT: '/support',
-	LOGS_SAVE_VIEWS: '/logs-save-views',
-	TRACES_SAVE_VIEWS: '/traces-save-views',
+	LOGS_SAVE_VIEWS: '/logs/saved-views',
```
Comment (YounixM) [inline frontend/src/constants/routes.ts:45]: Please add old - new route mapping to handle auto redirects for saved / old routes

**Review by YounixM — COMMENTED** [review]
Please add old - new route mapping to handle auto redirects for saved / old routes

File: frontend/src/constants/routes.ts (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/constants/routes.ts:46]: Done.

File: frontend/src/constants/routes.ts (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/constants/routes.ts:45]: Done.

**Review by YounixM — APPROVED** [review]

**Pull request merged**